packages/pretty-format/src/plugins/dom_element.js

```diff
@@ -78,24 +84,31 @@ export const serialize = (
     return printComment(node.data, config);
   }
 
-  const type = node.tagName.toLowerCase();
+  let type;
+
+  if (node.nodeType === FRAGMENT_NODE) {
+    type = `Fragment`;
```

Comment:
How do browsers show this type of node?

Reply:
Without any wrapper at all
![image](https://user-images.githubusercontent.com/1404810/42830782-b484afde-89ec-11e8-81e2-70da7c9c8f9b.png)

Reply:
So maybe a DocumentFragment? Fragment and <> look like from React

Reply:
done